src/js/org-counter.js

```
@@ -0,0 +1,64 @@
+
+  /**
+   * Update the organization count display based on filtered results
+   * @param {Array} filteredOrgs - Array of filtered organizations
+   * @param {Array} totalOrgs - Array of all organizations
+   */
+  updateCount(filteredOrgs, totalOrgs) {
+    this.filteredCount = filteredOrgs.length;
+    this.totalCount = totalOrgs.length;
+    
+    if (this.countElement) {
+      if (filterState.hasActiveFilters()) {
```

ESLint

[error] 'filterState' is not defined.

(no-undef)